### PR TITLE
Add API docs

### PR DESCRIPTION
**Pull request by karniv00l**
for repman-io/repman#270

Displaying swagger docs is handled, `https://app.repman.io/api/doc.js` will be used

Screenshot 2020-09-11 at 12 12 27

### CODE REVIEW

**Comment by karniv00l** [comment]
Displaying swagger docs is handled, `https://app.repman.io/api/doc.js` will be used

File: api.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Generate a new API Token
+
+In order to start using API you will need to generate `API Token`. In order to do that:
+
+![API Tokens menu](/docs/static/articles/user-profile/menu.png)
+
+1. Click `API Tokens` in user's menu
+1. Click `Generate a new API Token`
+1. Put token name and click `Generate`
+1. You will be redirected back to the list. Now you can copy your newly generated token and start using API.
```
Comment (akondas) [inline api.mdx:18]: Numbers?

File: api.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Generate a new API Token
+
+In order to start using API you will need to generate `API Token`. In order to do that:
+
+![API Tokens menu](/docs/static/articles/user-profile/menu.png)
+
+1. Click `API Tokens` in user's menu
+1. Click `Generate a new API Token`
+1. Put token name and click `Generate`
```
Comment (akondas) [inline api.mdx]: `Enter token name`

File: api.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+1. You will be redirected back to the list. Now you can copy your newly generated token and start using API.
+
+## Reference
```
Comment (akondas) [inline api.mdx:31]: Pleas add section about authorization:

## Authorization

In order to authorize the api you should use the `X-API-TOKEN` header with generated token. Example request:


```
curl --location --request GET 'https://repman.io/api/organization' \
--header 'X-API-TOKEN: d45d1eb9e4e9ab13fd8668557711bc81786a3ea60f5bb209581acf14ec036743'
```

File: api.mdx (hunk 1 above)
Comment (karniv00l) [reply api.mdx:18]: that's proper markdown

File: api.mdx (hunk 3 above)
Comment (karniv00l) [reply api.mdx:31]: done

**Review by akondas — APPROVED** [review]

**Review by akondas — APPROVED** [review]

**Pull request merged**